packages/manager/apps/pci-instances/public/translations/actions/Messages_fr_FR.json

```diff
@@ -15,6 +15,11 @@
   "pci_instances_actions_hard_reboot_instance_success_message": "Le redémarrage de l'instance {{ name }} a commencé.",
   "pci_instances_actions_hard_reboot_instance_title": "Redémarrer à froid (hard)",
   "pci_instances_actions_hard_reboot_instance_confirmation_message": "Vous allez redémarrer votre instance {{ name }}.",
+  "pci_instances_actions_reinstall_instance_title": "Réinstaller une instance",
+  "pci_instances_actions_reinstall_instance_confirmation_message": "Êtes-vous sûr de vouloir réinstaller l'instance {{ name }} ?",
+  "pci_instances_actions_reinstall_instance_warning_message": "Toutes les données seront supprimées.",
```

Comment:
Question: Could this one be used as a global message ? (is there another case that will consume this message?)